crml/sylo/src/e2ee.rs

```diff
@@ -50,6 +62,12 @@ decl_module! {
 			<PreKeyBundles<T>>::mutate((sender, device_id), |current_pkbs| current_pkbs.extend(pkbs));
 		}
 
+		/// Add a new PreKey bundle for a given user's device.
+		///
+		/// weight:
+		/// O(1) where n is the number of users and devices registered
```

Comment:
maybe out of date? `O(1) where n is the number of users and devices registered`

Reply:
yep out of dated. this is now fixed